common/header.html

```diff
@@ -53,10 +57,10 @@
     tagName: "div",
 
     html() {
-      if (tagged) {
-        return h("a.btn-default.custom-button", {href: url, title:"MUG Homepage", target: "_blank"}, "MUG Homepage");
-      } else {
+      if (!tagged) {
```

Comment:
Could you strengthen this further using strict equality comparison ?

Reply:
Resolved, will just commit changes